specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-01-01-preview/costmanagement.json

```diff
@@ -0,0 +1,1770 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01-preview",
+    "title": "CostManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CostManagement/exports": {
```

Comment:
The change in the PR is to rename reports resource type to export. 
resource type name of this entity seems like an action name instead of an entity name. As per service team, this is done to keep it in sync with UX. Ideally the UX should be changed to match with whatever is the most descriptive and meaningful name for this entity.